common/src/main/java/com/wynntils/features/user/CommandsFeature.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.wynntils.core.commands.ClientCommandManager;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.mc.event.ChatSentEvent;
+import com.wynntils.wc.utils.WynnUtils;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class CommandsFeature extends UserFeature {
```

Comment:
Also do we want this feature to be enabled/disabled by the user?

Reply:
Yes, I think it makes sense. Once again, if we do something bad. Or if, let's say Wynncraft introduces a `/feature` command, the user can disable this. Most of the features that the user can disable makes no real sense (in the end, if you disable all of them you might as well not use Wynntils :)) but I think it makes sense that each and every bit of modification we do can be individually turned off.